src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -221,6 +221,10 @@ describe('data', () => {
       filterData(DEFAULT_FILTERS).length
     );
   });
+  it('there is at least one disabled recommendation in the first page', () => {
+    const firstData = filterData({}).slice(0, DEFAULT_ROW_COUNT + 1);
```

Comment:
why `+1`? wouldn't it include one extra row from page 2?

Reply:
Yes, you are right :man_facepalming: